Library/ViewModels/LiveStreamChatViewModel.swift

```diff
@@ -225,6 +243,18 @@ LiveStreamChatViewModelOutputs {
           return Strings.Something_went_wrong_please_try_again()
         }
     }
+
+    self.chatInputViewMessageLengthCountLabelText = Signal.merge(
+      self.textProperty.signal,
+      self.viewDidLoadProperty.signal.mapConst("")
+      )
+      .map { $0.coalesceWith("") }
+      .map { "\($0.characters.count)/\(maxMessageLength)" }
```

Comment:
If I do as above and just have the number start at `140` decrement into negatives would localization still be necessary? I'm not sure what Twitter does.